install_config/persistent_storage/persistent_storage_glusterfs.adoc

```diff
@@ -138,19 +138,40 @@ variables begin with `openshift_storage_glusterfs_registry_`. A few other
 variables, such as `openshift_hosted_registry_storage_kind`, interact with the
 GlusterFS clusters.
 
-It is recommended to specify version tags for all containerized components.
-This is primarily to prevent components, particularly the {gluster} pods, from
-upgrading after an outage which may lead to a cluster of widely disparate
-software versions. The relevant variables are:
-
-- `openshift_storage_glusterfs_version`
-- `openshift_storage_glusterfs_block_version`
-- `openshift_storage_glusterfs_s3_version`
-- `openshift_storage_glusterfs_heketi_version`
-- `openshift_storage_glusterfs_registry_version`
-- `openshift_storage_glusterfs_registry_block_version`
-- `openshift_storage_glusterfs_registry_s3_version`
-- `openshift_storage_glusterfs_registry_heketi_version`
+It is recommended to specify the image name and version tags for all
```

Comment:
Maybe replace lines 141 - 144 with following:
```
To prevent {gluster} pods from upgrading after an outage leading to a cluster
with different {gluster} versions. It is recommended to specify the image name
and version tags for all containerized components. The relevant variables are:
```